src/bulk/common.ts

```diff
@@ -864,11 +808,21 @@ export interface BulkOperationPrivate {
 
 /** @public */
 export interface BulkWriteOptions extends CommandOperationOptions {
-  /** Allow driver to bypass schema validation in MongoDB 3.2 or higher. */
+  /**
+   * Allow driver to bypass schema validation in MongoDB 3.2 or higher.
+   * @defaultValue `false` - documents will be validated by default
+   **/
   bypassDocumentValidation?: boolean;
```

Comment:
```suggestion
  /**
   * Allow driver to bypass schema validation.
   * @defaultValue `false` - documents will be validated by default
   **/
  bypassDocumentValidation?: boolean;
```
we may as well clean up a reference to a long since supported server version